### ISSUE TITLE
Ubuntu 16.04 - No candidate version available for php5-fpm

### ISSUE THREAD

**Issue by freedev** [yevgenko/cookbook-php-fpm#93]
Moving from ubuntu 14.04 to 16.04 now the Vagrant provisioning of "recipe[php-fpm::default]" raise this exception:

Chef::Exceptions::Package: apt_package[php5-fpm] (php-fpm::install line 34) had an error: Chef::Exceptions::Package: No candidate version available for php5-fpm

If you need I can post the entire stacktrace.

**Comment by Thecakeisgit** [comment]
`php5` is deprecated in Ubuntu 16.04, however you could do this : 
```
sudo add-apt-repository ppa:ondrej/php
sudo apt update
sudo apt install php5.6-fpm
```
for php5.6.

**Comment by Thecakeisgit** [comment]
@freedev

**Comment by hrak** [comment]
This problem should be solved with the recent merge of PR #83

**Comment by yevgenko** [comment]
thanks for pointing this out @hrak, closing for now.
@freedev if it still an issue, just let me know, and thank you for reporting!
